### PR TITLE
Three reg page

### PR DESCRIPTION
**Pull request by SebastianGerS**
register page 
markup  style by It@Tokidaze 
I edited the Auth files for the backend to work, so now it's possible to register users
Also implemented a way to select whitch css file to load by looking at the curent route, how to make use of this is described in the readme file (when the new readme has been merged).

### CODE REVIEW

File: resources/views/auth/register.blade.php [hunk 1]
```diff
@@ -73,5 +74,85 @@
...
         </div>
     </div>
+</div-->
+<!-- endsection -->
+
+<div class="container" id="reg-container">
+
+    <h2 id="reg-h2">We are happy to have you here! Please fill out the following lines:</h2>
+
+    <!-- Form -->
```
Comment (axeberg) [inline resources/views/auth/register.blade.php:84]: I'm all for this during development but please remove all of these when you're finalizing stuff.

File: resources/views/layouts/header.blade.php [hunk 2]
```diff
@@ -1,3 +1,11 @@
+@php 
+
+    use Illuminate\Support\Facades\Route;
+
+    $routeName = Route::currentRouteName();
```
Comment (axeberg) [inline resources/views/layouts/header.blade.php:5]: Oh, clever. Please share this insight with the other teams who might be having problems with this.

**Review by axeberg — COMMENTED** [review]
👍  LGTM

**Review by AnnaRoss — APPROVED** [review]

File: resources/views/auth/register.blade.php (hunk 1 above)
Comment (Tokidaze) [reply resources/views/auth/register.blade.php:84]: Do you mean the comment or the H2?

**Pull request merged**